tests/rptest/tests/end_to_end.py

```diff
@@ -121,6 +123,24 @@ def start_redpanda(self,
             self.redpanda.restart_nodes(nodes_to_upgrade)
 
         self._client = DefaultClient(self.redpanda)
+        self._rpk_client = RpkTool(self.redpanda)
+
+    def create_topic(self, topic, partition, replication_factor):
```

Comment:
maybe we can make it a part of `RpkTool` ?

Reply:
ok, I've put it there